resources/views/myhub/company/details.blade.php

```diff
@@ -25,7 +25,7 @@
                         @livewire('company.manage-logo', ['company' => $company])
                         <x-section-border/>
                         @livewire('company.details-form', ['company' => $company ])
-                        @can('viewMembers', $company)
+                        @can('viewAnyMember', $company)
                         @livewire('company.member-manager', ['company' => $company])
```

Comment:
When member is invited and try to register via the email this error is thrown:
<img width="820" alt="image" src="https://github.com/HZ-HBO-ICT/it-conference/assets/77020646/1223d7c0-b930-433d-9fa8-b298cf9e9be8">

Edit: Saw the opened issue

Reply:
Wontfix then